### PR TITLE
CYC-64 Create InventoryItemController and update function

### LINKED ISSUE #60: CYC-64 Add update inventory controller function
## Description
[CYC-64](https://soen390team13.atlassian.net/browse/CYC-64)
Inventory controller should be able to handle update requests.

### PR DESCRIPTION
**Pull request by IanjhPhillips**
## Description
Creates InventoryItemController class with an update function to handle put requests to edit inventory items. 
Closes #60 

## Changes
* Create InventoryItemController.php
* Add update function to inventory item controller.
* Add route to web.php for update function.
* Modified InventoryItem model to guard id.

### CODE REVIEW

**Review by luigibesani — APPROVED** [review]
This works for me, I will add a route that generates csrf token.

File: app/Http/Controllers/InventoryItemController.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            "stock" => "nullable|integer|min:0",
+            "category" => "nullable|string|max:255",
+            "size" => "nullable|string|max:255",
+            "color" => "nullable|string|max:255",
+            "finish" => "nullable|string|max:255",
+            "material" => "nullable|string|max:255",
+            "part_number" => "nullable|string|max:255",
+            "lead_time" => "nullable|numeric|min:0",
+            "labour_cost" => "nullable|numeric|min:0"
+        ]);*/
```
Comment (m-triassi) [inline app/Http/Controllers/InventoryItemController.php:41]: I agree, this seems a little weird, we should spin up an issue for this at some point

**Comment by IanjhPhillips** [comment]
lgtm

**Review by m-triassi — APPROVED** [review]
Nice work guys, this is working perfectly!

**Pull request merged**